src/device-registry/utils/create-site.js

```diff
@@ -775,80 +791,83 @@ const manageSite = {
             success: true,
             message: "successfully retrieved the altitude details",
             data: r.data.results[0].elevation,
+            status: HTTPStatus.OK,
           };
         })
         .catch((e) => {
-          logElement("get altitude server error", e.message);
+          logElement("get altitude server error", { message: e.message });
           return {
             success: false,
             message: "get altitude server error",
-            error: e,
+            errors: { message: e },
+            status: HTTPStatus.BAD_GATEWAY,
           };
         });
     } catch (e) {
-      logElement("server error", e.message);
+      logElement("server error", { message: e.message });
       return {
         success: false,
         message: "get altitude server error",
-        error: e.message,
+        errors: { message: e.message },
+        status: HTTPStatus.INTERNAL_SERVER_ERROR,
       };
     }
   },
 
   getTrafficFactor: (lat, long) => {
     try {
     } catch (e) {
-      logElement("server error", e.message);
+      logElement("server error", { message: e.message });
     }
   },
 
   getGreenness: (lat, long) => {
     try {
     } catch (e) {
-      logElement("server error", e.message);
+      logElement("server error", { message: e.message });
     }
   },
 
```

Comment:
@Baalmart , why do we have empty try statements yet this information can be gotten from the data warehouse???